parquet2hive/parquet2hive

```diff
@@ -147,6 +147,10 @@ def transform_type(avro):
         record = "struct<{}>".format(fields_decl)
         udf[avro["name"]] = record
         return record
+    elif isinstance(avro, dict) and avro["type"] == "struct":
```

Comment:
Could we just `or` the condition with the branch of the `record` type since the code is the same?